compiler/luci/service/src/CircleShapeInferenceRule.cpp

```diff
@@ -1121,6 +1121,40 @@ class ShapeInferenceAlgorithm final : public luci::CircleNodeVisitor<loco::NodeS
     return loco::NodeShape{output_shape};
   }
 
+  loco::NodeShape visit(const luci::CirclePadV2 *node) final
+  {
+    const loco::DataType S32 = loco::DataType::S32;
```

Comment:
nitpick
In this situation it is better to use `constexpr`, because compiler has more guarantees for optimizations.

Reply:
I looked at code around, there are no `constexpr`s, so I'll leave this piece as-is.